File: TenMinsJava/day36.java
// get remaining milliseconds of current the day
package TenMinsJava;

import java.time.LocalTime;

public class day36 {
    public static void main(String[] args) {
        LocalTime t = LocalTime.now();
        long ms = t.toNanoOfDay();
        System.out.println(86400000L - ms);
    }
}

File: TenMinsPy/day36.py
# get remaining milliseconds of current the day

from datetime import datetime

class day36:
    def __init__(self):
        n = (datetime.now().hour * 3600000) + (datetime.now().minute * 60000) + (datetime.now().second * 1000) + (datetime.now().microsecond // 1000)
        print(86400000 - n)

day36()